easydigitaldownloads/easy-digital-downloads
# Revise upgrade notice

Fixes #7834

Proposed Changes:
Updates the upgrade notice output and wording for improved accessibility and consistency with the notice on the upgrade page.

To test:
On a site which has EDD 2.x data (even if a migration has been run and the legacy data has not been deleted), add a random array key/value to the [`$upgrades` array](https://github.com/easydigitaldownloads/easy-digital-downloads/blob/issue/7834/includes/admin/upgrades/upgrade-functions.php#L205) to expose the admin notice.
